scripts/bootstrap/buildbootstrapcli.sh

```diff
@@ -193,7 +193,7 @@ echo "**** DETECTING GIT COMMIT HASHES ****"
 # the seed cli package was built from
 __coreclrhash=`strings $__seedclipath/shared/Microsoft.NETCore.App/$__frameworkversion/libcoreclr.so | grep "@(#)" | grep -o "[a-f0-9]\{40\}"`
 __corefxhash=`strings $__seedclipath/shared/Microsoft.NETCore.App/$__frameworkversion/System.Native.so | grep "@(#)" | grep -o "[a-f0-9]\{40\}"`
-__coresetuphash=`strings $__seedclipath/dotnet | grep -o "[a-f0-9]\{40\}"`
+__coresetuphash=`strings $__seedclipath/dotnet | grep "@(#)" | grep -o "[a-f0-9]\{40\}"`
```

Comment:
`strings $__seedclipath/dotnet` is returning two strings on tip.

Reply:
With this change, the script would not be usable for the 2.0.x branch and to enable managed part of the build to just to work in our current coreclr, corefx and core-setup branches, we need to create 2.0.x bootstrap cli, since that's still being used for the builds (my educated guess is that it will be that way at least until the 2.1.x is officially released).
So we should handle both cases here - first try this new way and if it doesn't extract any hash, use the old one.

Reply:
Done